galaxy.yml

```diff
@@ -19,5 +19,15 @@ tags:
 - cisco
 version: 1.3.1
 build_ignore:
-- tests/output/*
+- 'tests'
 - '*tar.gz'
+- '*.DS_Store'
+- 'CHANGELOG.md'
+- '*.json'
+- 'venv'
+- '.vscode'
+- '.gitignore'
+- '.env'
+- '.github'
+- 'changelogs'
+- 'README.md'
```

Comment:
hmm... a few things I note about this list.....
(1) Are you sure you want to remove CHANGELOG.md and README.md?  Those seem useful to the end user and might be used by other tools (for isntance, I think galaxy will parse that for displaying its description?)
(2) I'm not sure whether removing tests and changelogs is okay... (I am not a lawyer.... but I do have a question in to our legal team about exactly this issue...) Since this is licensed GPLv3, the collection source needs to be available.  Removing changelogs and tests may make it so the collection tarball does not count as source.  If that's the case, it becomes trickier whether the ansible-2.10 tarball can include this collection.

(all of this is stuff we're having to ask the lawyers about).

Reply:
Please let me know what the answer is on the tests and changelog fragments. I'm fine including CHANGELOG.md and README.md and have fixed those.

Reply:
I would personally keep them, they take very little amount of space and are useful for the end user to see.

Reply:
Just pushed an update which includes tests and changelog fragments. Specifically it's just excluding `tests/output`.